src/librustc_resolve/build_reduced_graph.rs

```diff
@@ -108,20 +116,20 @@ impl<'b> Resolver<'b> {
                 let module_path: Vec<Name> = match view_path.node {
                     ViewPathSimple(_, ref full_path) => {
                         full_path.segments
-                                 .split_last()
-                                 .unwrap()
-                                 .1
-                                 .iter()
-                                 .map(|seg| seg.identifier.name)
-                                 .collect()
+                            .split_last()
+                            .unwrap()
+                            .1
+                            .iter()
+                            .map(|seg| seg.identifier.name)
+                            .collect()
                     }
 
                     ViewPathGlob(ref module_ident_path) |
                     ViewPathList(ref module_ident_path, _) => {
                         module_ident_path.segments
-                                         .iter()
-                                         .map(|seg| seg.identifier.name)
-                                         .collect()
+                            .iter()
+                            .map(|seg| seg.identifier.name)
+                            .collect()
```

Comment:
Advantage here is that, if the binding name length changes, those methods would not need re-indenting